src/TraceEvent/TraceEvent.cs

```diff
@@ -1190,6 +1190,7 @@ public string Dump(bool includePrettyPrint = false, bool truncateDump = false)
             XmlAttrib(sb, "Version", Version);
             XmlAttribHex(sb, "Keywords", (ulong)Keywords);
             XmlAttrib(sb, "TimeStampQPC", TimeStampQPC);
+            sb.Append(" QPC=\"").Append((1000000.0 / Source.QPCFreq).ToString("f3")).Append("us\"");
```

Comment:
Nit: Perhaps `QPF=` or `QPCFreq=` would be better since it's the frequency, not the count.